libs/core/langchain_core/runnables/fallbacks.py

```diff
@@ -454,43 +455,44 @@ def stream(
             name=config.get("run_name"),
             run_id=config.pop("run_id", None),
         )
-        first_error = None
         last_error = None
+        output: Optional[Output] = None
         for runnable in self.runnables:
+            stream = runnable.stream(
+                input,
+                patch_config(config, callbacks=run_manager.get_child()),
+                **kwargs,
+            )
+
+            chunk = None
             try:
                 if self.exception_key and last_error is not None:
                     input[self.exception_key] = last_error
-                stream = runnable.stream(
-                    input,
-                    patch_config(config, callbacks=run_manager.get_child()),
-                    **kwargs,
-                )
-                chunk = next(stream)
+
+                for chunk in stream:
+                    yield cast(Output, chunk)
+
+                    if isinstance(chunk, RemoveMessage):
```

Comment:
I think `__radd__` of RemoveMessageChunk should implement this behavior no? Otherwise what's the point of the chunk variant

Reply:
which behavior?